dvc/remote/gdrive.py

```diff
@@ -248,7 +291,11 @@ def _ids_cache(self):
         cache = {
             "dirs": defaultdict(list),
             "ids": {},
-            "root_id": self._get_item_id(self.path_info, use_cache=False),
+            "root_id": self._get_item_id(
+                self.path_info,
+                use_cache=False,
+                hint="Check remote directory exists and you can access it.",
```

Comment:
"remote location", "directory in remote URL: {}" ?

Reply:
updated a bit